homeassistant/components/airly/__init__.py

```diff
@@ -115,3 +126,13 @@ def __init__(self, session, api_key, latitude, longitude):
         except (ValueError, AirlyError, ClientConnectorError) as error:
             _LOGGER.error(error)
             self.data = {}
+
+        # Airly allows 100 requests per day. We check how many Airly config entries are
+        # and calculate interval to not exceed allowed numbers of requests.
+        instances = len(self._hass.config_entries.async_entries(DOMAIN))
+        interval = ceil(24 * 60 / MAX_REQUESTS_PER_DAY) * instances
```

Comment:
Shouldn't we treat 24*60 as magic numbers?

Reply:
You mean swapping these numbers to constants?